contrib/py/admin/dumpstats.py

```diff
@@ -0,0 +1,166 @@
+#!/usr/bin/env python3
+import requests
+import json
+import time
+import curses
+
+class Canvas():
+
+  def __init__(self, url):
+    self.data = dict()
+    self.win = curses.initscr()
+    self._url = url
+
+  def __del__(self):
+    self.win.clear()
+    curses.endwin()
+
+  def on_timer(self, event):
+    """called on timer event"""
+    self.update_data()
+
+  def _log(self, *args):
+    print("{}".format(*args))
+
+  def jsonrpc(self, meth, params):
+    r = requests.post(self._url, headers={"Content-Type": "application/json"}, json={"jsonrpc": "2.0", "id": '0', "method":'{}'.format(meth), "params": params})
+    return r.json()
+
+  def update_data(self):
+    """update data from lokinet"""
+    try:
+      j = self.jsonrpc("llarp.admin.dumpstate", {})
+      self.data = j['result']
+    except:
+      self.data = None
+
+  def _render_path(self,y, path, name):
```

Comment:
Probably not worth it at this scale, but `mypy` style typings could be useful here (since I have no idea what `y`, `path` or `name` are meant to be)

Reply:
yeah this was just a quick weekend project to do